Probesys/bileto
# dev: Refactor the handling of MessageDocuments

## Related issue(s)



https://github.com/Probesys/bileto/issues/149

## Changes



See commits

## How to test manually



- upload an image
- refresh the page → check it still appears under the editor
- post the message → check it's attached to the message
- upload an image
- detach/remove the attachment → check it's removed from the list and from the editor

## Checklist
